client/i18n/locales/japanese/links.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Any of the links in this file that have been translated to Japanese can be replaced with the appropriate URL

Reply:
I updated some links which already have Japanese version, and the /news site which we would soon be avaiable.